src/vm_cpu.rs

```diff
@@ -820,10 +817,10 @@ impl<'vcpu, 'pages, 'host, T: GuestStagePagingMode> ActiveVmCpu<'vcpu, 'pages, '
 
         use VmExitCause::*;
         match cause {
-            ResumableEcall(msg) | FatalEcall(msg) | BlockingEcall(msg, _) => {
+            ResumableEcall(msg) | FatalEcall(msg) => {
                 self.report_ecall_exit(msg);
             }
-            ForwardedEcall(msg) => {
+            ForwardedEcall(msg) | BlockingEcall(msg) => {
```

Comment:
i think we can totally get rid of the `BlockingEcall` exit cause, right?

Reply:
We use it for setting up the `pending_op`. Should I replace it with `ForwardedEcall` so that I can fully remove it?

Reply:
As per our offline conversation, I've replaced BlockingEcall with ForwardedEcall.